Configuration.sln

```diff
@@ -41,6 +41,10 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Configu
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Configuration.Binder.Test", "test\Microsoft.Framework.Configuration.Binder.Test\Microsoft.Framework.Configuration.Binder.Test.xproj", "{AE6FFE9B-6378-4D57-AA24-7D257F18B235}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Configuration.File", "src\Microsoft.Framework.Configuration.File\Microsoft.Framework.Configuration.File.xproj", "{881E7CBC-492C-47C5-98A6-61DD1C753EE6}"
```

Comment:
Hmm, a name for a package with one type that has 2 or 3 methods in it... :smile: 

`Microsoft.Framework.Configuration.FileHelpers` ? Kind of a lame name, but that's all these are. None are absolutely required to be used (anyone can easily write the code for what they do).